exporters/zipkin/include/opentelemetry/exporters/zipkin/recordable.h

```diff
@@ -67,6 +71,7 @@ class Recordable final : public sdk::trace::Recordable
 
 private:
   ZipkinSpan span_;
+  std::string service_name_;
```

Comment:
I'm thinking here about some optimization opportunity: how often do you expect `service_name` value to change, i.e. do you have to copy it here for every recordable, then again into every `JSON` (ZipkinSpan) object?

Reply:
We can think of some optimization of not passing resource and instrumentation library as part of every span, but still, these data need to be copied to every ZipkinSpan. ( https://zipkin.io/zipkin-api/#/default/post_spans ).

Reply:
That's fine for now. I am looking at Zipkin exporter as example for Fluentd exporter, I can share some thoughts how I would've rearranged this in the Fluentd exporter.

Reply:
Thanks, will look forward to that :)